.github/workflows/self-push.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - master
-      - model-templates
+      - ci_*
```

Comment:
That's just for testing, right?

Reply:
No, that's so that when branches contain the `ci_*` wildcard in their names, the CI is run on them. It's currently the same on the scheduled run.

This allows to easily test branches, but I think it's smarter to have it on these GPU tests than on the scheduled tests, which take ~8 hours to run all the tests (and that runs on each commit!).

The previous code `model-templates` was only for testing purposes though, and shouldn't have been merged.

Reply:
I'd vote for removing that `ci_*` on the scheduled run but keeping it here.